x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/log_rate_analysis_results_table_groups.tsx

```diff
@@ -93,6 +93,11 @@ export const LogRateAnalysisResultsGroupsTable: FC<LogRateAnalysisResultsTablePr
 
   const pinnedGroup = useAppSelector((s) => s.logRateAnalysisTableRow.pinnedGroup);
   const selectedGroup = useAppSelector((s) => s.logRateAnalysisTableRow.selectedGroup);
+  const {
+    analysisType,
+    windowParameters,
+    documentStats: { documentCountStats },
+  } = useAppSelector((s) => s.logRateAnalysis);
```

Comment:
Since `useColumns` is a hook, you could move this inside the hook itself so doesn't need to get passed on as arguments to `useColumns`.

Reply:
Good catch - moved in ff861eb3f280d3812b7536fcbabcd06eced039cc